test/mobile/test_lite_script_module.py

```diff
@@ -63,6 +63,33 @@ def forward(self, x):
         assert(b"top(B).A0(A).forward" in bytes)
         assert(b"top(B).A1(A).forward" in bytes)
 
+    def test_save_mobile_module_with_debug_info_with_script(self):
+        class A(torch.nn.Module):
+            def __init__(self):
+                super(A, self).__init__()
+
+            def forward(self, x):
+                return x + 1
+
+        class B(torch.nn.Module):
+            def __init__(self):
+                super(B, self).__init__()
+                self.A0 = A()
+                self.A1 = A()
+
+            def forward(self, x):
+                return self.A0(x) + self.A1(x)
+
+        input = torch.tensor([5])
+        trace_module = torch.jit.script(B(), input)
+        bytes = trace_module._save_to_buffer_for_lite_interpreter(_save_mobile_debug_info=True)
+
+        assert(b"mobile_debug.pkl" in bytes)
+        assert(b"module_debug_info" in bytes)
+        assert(b"top(B).forward" in bytes)
+        assert(b"top(B).A0(A).forward" in bytes)
+        assert(b"top(B).A1(A).forward" in bytes)
+
```

Comment:
Can you also add testcase for:
```
import torch
import torch.nn as nn

class A(nn.Module):
  def __init__(self):
    super(A, self).__init__()

  def forward(self, x):
    return x + 1

class B(nn.Module):
  def __init__(self):
    super(B, self).__init__()

  def forward(self, x):
    return x + 2

class C(nn.Module):
  def __init__(self):
    super(C, self).__init__()
    self.A0 = A()
    self.B0 = B()

  def forward(self, x):
    return self.A0(self.B0(x)) + 1
```
You should still expect to see (top(C).B0(B).forward) and (top(C).A0(A).forward)